evap/results/templates/results_index.html

```diff
@@ -81,15 +81,15 @@ <h6 class="card-subtitle mb-1">{% trans 'Semester' %}</h6>
                         </tr>
                     </thead>
                     <tbody>
-                    {% regroup evaluations by course as course_evaluations %}
-                    {% for course, evaluations in course_evaluations|dictsort:"grouper.name" %}
-                        {% if course.evaluations.count > 1 %}
+                    {% for course, evaluations in courses %}
+                        {# this is the reason why results_index_evaluation does not need to include is_subentry in the cache key: an evaluation is a subentry or no subentry regardless of which user asks #}
+                        {% if course.num_evaluations > 1 %}
                             {% include 'results_index_course.html' %}
-                            {% for evaluation in evaluations|dictsort:"name" %}
+                            {% for evaluation in evaluations %}
                                 {% include 'results_index_evaluation.html' with links_to_results_page=evaluation|can_results_page_be_seen_by:request.user is_subentry=True %}
                             {% endfor %}
                         {% else %}
-                            {% for evaluation in evaluations|dictsort:"name" %}
```

Comment:
@janno42 can you tell me why those sorts were here? perhaps to match what JS will sort by default so the user sees no change?